### DISCUSSION TITLE
What's the difference between deepl free and deeplx

### DISCUSSION THREAD

**Discussion by ghost** [pot-app/pot-desktop#527] [Q&A] [answered]
i asked it because deeplx does free deepl api too, so what's the difference between of them 5000 word limit or sum.?

**Comment by Pylogmon** [comment, marked as answer]
In fact, there is no difference. The deepl free built into pot actually uses the same implementation as deeplx. The only difference is that you do not need to start a deeplx service.

**Reply by ghost** [reply]
@Pylogmon thank you so much! also can you explain differences between incremental and dynamic mode?

**Reply by Pylogmon** [reply]
Incremental translation: Without closing the translation window, select the words again for translation, and the selected content will be appended to the previous content.
Dynamic translation: refers to automatically triggering translation when typing a translation without having to click the translation button or press Enter.
